src/Twig/Extension/StfalconTinymceExtension.php

```diff
@@ -67,6 +67,13 @@ public function getParameter($name)
      */
     public function getFunctions()
     {
+        if(class_exists('\Twig_SimpleFunction')){
```

Comment:
With the version constraints above this makes no sense, as ``Twig_SimpleFunction`` exists since 1.12.0